Habitica/res/menu/chat_message.xml

```diff
@@ -10,6 +10,11 @@
         android:title="@string/send_pm" />
     <group>
         <item
+                android:id="@+id/menu.chat.copy"
+                android:icon="@drawable/ic_action_warning"
+                app:showAsAction="always|withText"
+                android:title="@string/copy_chat_message" />
```

Comment:
This string isn't found in the strings.xml file. did you maybe forget to commit it?